# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jack/powerpc64.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- jack-audio-connection-kit-0.109.2/config/sysdeps/atomicity.h.vanilla	2008-07-20 13:43:58.000000000 +0200
+++ jack-audio-connection-kit-0.109.2/config/sysdeps/atomicity.h	2008-07-20 13:44:55.000000000 +0200
@@ -11,7 +11,7 @@
 
 #include <config/cpu/i386/atomicity.h>
 
-#elif defined(__powerpc__) || defined(__ppc__) /* linux and OSX use different tokens */
+#elif (defined(__powerpc__) || defined(__ppc__)) && !defined(__powerpc64__) /* linux and OSX use different tokens */
 
 #include <config/cpu/powerpc/atomicity.h>
 
--- jack-audio-connection-kit-0.109.2/config/sysdeps/cycles.h.vanilla	2008-07-20 13:44:59.000000000 +0200
+++ jack-audio-connection-kit-0.109.2/config/sysdeps/cycles.h	2008-07-20 13:45:13.000000000 +0200
@@ -15,7 +15,7 @@
 
 #include <config/cpu/i486/cycles.h>
 
-#elif defined(__powerpc__) || defined(__ppc__)   /* linux and OSX gcc use different tokens */
+#elif (defined(__powerpc__) || defined(__ppc__)) && !defined(__powerpc64__) /* linux and OSX gcc use different tokens */
 
 #include <config/cpu/powerpc/cycles.h>
 
